lib/generators/boring/whenever/install/install_generator.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Boring
+  module Whenever
+    class InstallGenerator < Rails::Generators::Base
+      desc "Adds whenever gem to the application for managing cron jobs"
+
+      def add_whenever_gem
+        say "Adding whenever gem", :green
+
+        Bundler.with_unbundled_env do
+          run "bundle add whenever"
+        end
+      end
+
+      def run_whenever_generator
+          say "Running whenever generator", :green
```

Comment:
Let's update the description to "Create schedule.rb file"

You can ask for PR review by mentioning Abhay after you have resolved these new comments.